src/main/java/net/sf/jabref/gui/event/listener/EventListener.java

```diff
@@ -0,0 +1,11 @@
+package net.sf.jabref.gui.event.listener;
+
+import com.google.common.eventbus.Subscribe;
+
+import net.sf.jabref.gui.event.Event;
+
+public interface EventListener {
```

Comment:
Should probably inherit from https://docs.oracle.com/javase/7/docs/api/java/util/EventListener.html